StratisCore.UI/src/app/wallet/send/send.component.ts

```diff
@@ -45,7 +45,7 @@ export class SendComponent implements OnInit, OnDestroy {
   public apiError: string;
   public firstTitle: string;
   public secondTitle: string;
-  public opReturnAmount: number = 0;
+  public opReturnAmount: number = 100000000;
```

Comment:
Is this one entire coin? Why is it so high?

Reply:
It should be 1 sat, not 1 coin, I misread the requirement for the story. Should be ok now.